Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "drawbridge"
+version = "0.1.0"
+authors = ["The Enarx Project Developers"]
+edition = "2021"
+
+[dependencies]
+axum = "0.4.5"
+tower = "0.4.12"
+
+auth-service = { path = "./src/service/auth" }
+object-store = { path = "./src/service/object-store" }
+tag-store = { path = "./src/service/tag-store" }
```

Comment:
If we're doing separate crates, this should probably be a workspace. As for names: `auth`, `tree`, `tags`.